modules/oe_theme_demo/README.md

```diff
@@ -10,7 +10,6 @@ The demo module includes:
 - Placeholder blocks like:
  - Language switcher
  - Site switcher
- - Search block
  - Footer
```

Comment:
The footer is also not provided by the demo module anymore right?

Reply:
yes. Fixed.